app/src/pages/inside/stepPage/modals/postIssueModal/utils.js

```diff
@@ -0,0 +1,36 @@
+export const validate = (fields, validationConfig) => {
+  let validValues = {
+    username:
+      !fields.username && typeof fields.username === 'string' ? 'requiredFieldHint' : undefined,
+    password:
+      !fields.password && typeof fields.password === 'string' ? 'requiredFieldHint' : undefined,
+    accessKey:
+      !fields.accessKey && typeof fields.accessKey === 'string' ? 'requiredFieldHint' : undefined,
+  };
+  if (validationConfig) {
+    const validatedFields = {};
+    Object.keys(validationConfig).map((key) => {
+      if (validationConfig[key]) {
+        validatedFields[key] = !fields[key] || !fields[key][0] ? 'requiredFieldHint' : undefined;
+      }
+      return null;
+    });
+
+    validValues = Object.assign(validValues, validatedFields);
+  }
+  return validValues;
+};
+
+export const createFieldsValidationConfig = (fields) =>
+  fields.reduce((acc, item) => ({ ...acc, [item.id]: item.required }), {});
+
+export function getSessionStorageItem(key) {
```

Comment:
Why do we need session storage?

Reply:
We discuss it earlier but I don't have any reasons nowadays. Replaced with `localStorage`